mappings/base_mapping.py

```diff
@@ -29,6 +29,21 @@ def applyMapping(self):
 
     def applyFormatting(self):
         return self.record
+    
+    def format_identifiers(self, identifiers):
+        for index, identifier in enumerate(identifiers):
+            count_digits = 0
+            if '/' not in identifier:
+                for char in identifier:
+                    if char.isdigit():
+                        count_digits += 1
+                if count_digits == 8:
+                    identifiers[index] = f'{identifier}|issn'
+                if count_digits == 10 or count_digits == 13:
+                    identifiers[index] = f'{identifier}|isbn'
+            else:
```

Comment:
CLACSO specific logic should not be included in this. 

What are your thoughts on the following to clean up this method a bit: 
- Using the sum function for counting the number of digits with an iterable through the identifier string
- Using a dictionary instead of multiple if statements for determining the proper identifier type (ISSN/ISBN) and passing that through the f string

Reply:
I'll implement the dictionary approach you suggested. I'll add  the '|clacso' string to the CLACSO mapping instead so all identifiers have that type by default.